src/App.scss

```diff
@@ -0,0 +1,235 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@300;400;500;700&display=swap');
+
+$theme-color: #0290ff;
+$base-color: #121212;
+$ll-gray: #e8e8e8;
+$gray-border: 1px solid $ll-gray;
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Montserrat, sans-serif;
+}
+
+body {
+  color: $base-color;
+}
+
+ul {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+.roboto {
+  font-family: 'Roboto Slab', serif;
+}
+
+.lb-text {
+  font-family: 'Roboto Slab', serif;
```

Comment:
- [ ] Great job adding variables to avoid repetition, I think if you can also extract this property `font-family: 'Roboto Slab', serif;` into a separate class or variable could be a good idea since it seems to be repeated multiple times 😉